source/mainnet/net/guides/recover-wallet.rst

```diff
@@ -47,3 +62,9 @@ In case you get a new device or need to restore your existing device, you can re
 .. |morepage| image:: ../images/more-ellipsis.png
              :alt: Three dots button
              :width: 40px
+.. |moredetails| image:: ../images/more-arrow.png
```

Comment:
There is a note 5 lines above this:
>"When you recover your wallet, any account names that you might have edited will be reset to the account number. You can :ref:`edit the account name<change-mw-acct-name>`, if desired."

This should probbably be moved into the mw's tab. (because that won't happen in the bw, and you can't edit the names of the accounts in the initial version)